spec/mailers/previews/bulk_url_upload_results_mailer_preview.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/bulk_url_upload_results_mailer
```

Comment:
Per our conversation, since the mailer previews work, I'm leaving them alone. I've no objection to using `mailcatcher` for future development. Also note that the rename of the system mailer layout from 'text.hml' to 'text.haml' means that you can now use the previews to look at the text version of the notification emails.